lit.go

```diff
@@ -109,8 +110,13 @@ func setConfig(lc *LitConfig) {
 }
 
 func main() {
-	fmt.Printf("lit node v0.0\n")
-	fmt.Printf("-h for list of options.\n")
+	f, err := os.OpenFile("logfile", os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	defer f.Close()
+	mw := io.MultiWriter(color.Output, f)
```

Comment:
We don't want colorized text in log file output; color can stay in the interactive ui only.